env/flink/env_flink_test_suite.cc

```diff
@@ -45,22 +55,118 @@ void EnvFlinkTestSuites::setUp() {
   }
 }
 
-void EnvFlinkTestSuites::testFileExist() {
-  std::string fileName("test-file");
-  Status result = flink_env_->FileExists(fileName);
-  ASSERT_TRUE(result.IsNotFound());
+void EnvFlinkTestSuites::testDirOperation() {
+  const std::string dir_name = "test-dir";
+  ASSERT_TRUE(flink_env_->FileExists(dir_name).IsNotFound());
+  ASSERT_TRUE(flink_env_->CreateDir(dir_name).ok());
+  ASSERT_TRUE(flink_env_->CreateDirIfMissing(dir_name).ok());
+  ASSERT_TRUE(!flink_env_->CreateDir(dir_name).ok());
```

Comment:
nit: How about introducing the "ASSERT_FALSE" macro definition?
For example: 
`#define ASSERT_FALSE(condition) ASSERT_TRUE(!condition)`

Reply:
Thanks for the suggestion.
I have modified accordingly.